Makefile

```diff
@@ -46,16 +49,19 @@ SED?=$(shell which gsed 2>/dev/null || which sed)
 # TODO(pintohutch): this is a bit hacky, but can be useful when testing.
 # Ultimately this should be replaced with go templating.
 define update_manifests
-	find manifests examples -type f -name "*.yaml" -exec sed -i "s#image: .*/$(1):.*#image: ${IMAGE_REGISTRY}/$(1):${TAG_NAME}#g" {} \;
+	find manifests examples -type f -name "*.yaml" -exec sed -i "s#image: .*/$(1):.*#image: ${IMAGE_REGISTRY}/$(1):$(TAG_NAME)#g" {} \;
 endef
 
 define docker_build
 	DOCKER_BUILDKIT=1 docker build --label "part-of=gmp" $(1)
 endef
 
-define docker_tag_push
-	docker tag $(1) $(2)
-	docker push $(2)
+define docker_tag
```

Comment:
Suggestion: Does using `define` this add any value when the command is one line? Maybe just use the command directly.

Reply:
Good point!